tests/TestHelpers/GraphHelper.php

```diff
@@ -97,6 +97,21 @@ public static function getFullUrl(string $baseUrl, string $path): string {
 		return $fullUrl;
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $path
+	 *
+	 * @return string
+	 */
+	public static function getBeta1FullUrl(string $baseUrl, string $path): string {
```

Comment:
if beta1 changes to beta2 and so on then it will help
```suggestion
	public static function getBetaFullUrl(string $baseUrl, string $path): string {
```